Jenkinsfile.integration

```diff
@@ -199,6 +220,9 @@ pipeline {
         stage('Deploy Airship') {
             options {
                 timeout(time: 45, unit: 'MINUTES', activity: true)
+                lock(label: reserve_env == 'true' ? cloud_env:'dummy-resource',
```

Comment:
We need to rebase as tempest step is added so we need to add this lock logic there as well.